src/Compilers/VisualBasic/Portable/Symbols/TypeSymbolExtensions.vb

```diff
@@ -16,6 +16,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         End Function
 
         <Extension()>
+        Public Function IsNullableType(this As ITypeSymbol) As Boolean
+            Dim original = this.OriginalDefinition
+
+            Return original IsNot Nothing AndAlso original.SpecialType = SpecialType.System_Nullable_T
+        End Function
```

Comment:
We should avoid extension methods on `ITypeSymbol`. There's an existing extension method on `TypeSymbol` and it looks like the callers above can use that relatively easily.